renovate.json

```diff
@@ -2,7 +2,10 @@
   "extends": [
     "config:base"
   ],
-  "schedule": [
-    "every 6 months"
-  ]
```

Comment:
We do want to have a schedule of some sort and not just bumping patch versions of dependencies. That was the original reason for renovate here.

Can we have some sort of schedule to prevent too many trivial PRs?

Reply:
If we move towards using the `^` in versions here (as I started to do),  you'll only get renovate PRs for semver major releases, and a weekly lock file update.  Those tend to be fairly infrequent.